app/src/ui/acknowledgements/acknowledgements.tsx

```diff
@@ -34,10 +34,21 @@ export class Acknowledgements extends React.Component<IAcknowledgementsProps, IA
   public constructor(props: IAcknowledgementsProps) {
     super(props)
 
-    this.state = { licenses: null }
+    this.state = { licenses: null, licenseText: null }
   }
 
   public componentDidMount() {
+    const licensePath = Path.join(getAppPath(), 'static', 'LICENSE')
```

Comment:
Our license dump actually already includes our license, we explicitly skip it: https://github.com/desktop/desktop/pull/1611/files#diff-bbe735d95bc2a145bffe0c707c9b00a4L62